1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,34 @@
-function setAlarm() {}
+function setAlarm() {
```

Comment:
The variables on line 4 and 7 are unused here, in vscode they should be slightly greyed out with a popup message of `is declared buts its values is never read`.